db/migrate/20180926182941_create_books.rb

```diff
@@ -1,12 +1,13 @@
 class CreateBooks < ActiveRecord::Migration[5.1]
   def change
+    
     create_table :books do |t|
-      t.string :genero
-      t.string :autor
-      t.string :imagen
-      t.string :titulo
+      t.string :genre
```

Comment:
I think almost every field is mandatory. If that's the case, please add `null: false` to be sure your data is consistent and to allow the DB engine to run the appropriate validations.